src/components/ProjectCard/ProjectCard.module.css

```diff
@@ -21,7 +21,8 @@
   height: 5rem;
   display: flex;
   flex-direction: column;
-  background-color: #f6fbe5;
+  /* background-color: #f6fbe5; */
+  background-color: #f1f1f1;
```

Comment:
@neelxie while changing to CSS modules I noticed that this component was using the CSS of another (MemberCard) and that's where it was picking the background color that I added here. I could not locate it, kindly advise.

Reply:
@fel-w  whatever you do for now, do not modularize the project card as it sprews a lot of other unknown bugs such as this. Work on your task without changing that file. ![Screenshot from 2021-08-16 09-31-15](https://user-images.githubusercontent.com/32802973/129520756-ad6fbe8c-52e2-488f-9d02-b1ffbf14e7a6.png)

Reply:
Noted, let me revert